registry/handlers/app.go

```diff
@@ -686,7 +686,7 @@ func (app *App) authorized(w http.ResponseWriter, r *http.Request, context *Cont
 			// Add the appropriate WWW-Auth header
 			err.SetHeaders(w)
 
-			if err := errcode.ServeJSON(w, errcode.ErrorCodeUnauthorized.WithDetail(accessRecords)); err != nil {
+			if err := errcode.ServeJSON(w, errcode.ErrorCodeUnauthorized.WithDetail(err.Error())); err != nil {
```

Comment:
I was expecting the authentication challenge error to be able to describe itself, but yeah, this change does not belong in this PR. I'll move it to the next one.